src/test-apps/happy/tests/standalone/wdmNext/test_weave_wdm_next_disable_notify_01.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+
+
+#
+#    Copyright (c) 2016-2017 Nest Labs, Inc.
```

Comment:
Consider to update copyright data for these changed/added file?

Reply:
Done